### PR TITLE
Wherigo: display warning when mapping zones and unified map is disabled

### PR DESCRIPTION
**Pull request by eddiemuc**
Wherigo: display warning when mapping zones and unified map is disabled

### CODE REVIEW

File: main/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -3013,6 +3013,8 @@
     <string name="wherigo_otm_shortcuts_title">Wherigo Player Shortcuts</string>
     <string name="wherigo_otm_shortcuts_message">If you want to use c:geo Wherigo Player regularly, consider creating a quick launch button or custom menu item for it under Settings -> Appearance</string>
 
+    <string name="wherigo_map_supported_for_unified_map_only">Displaying of Wherigo locations on map is not supported for your map settings. Please switch to unified map under Settings -> Map Sources -> Unified Map (beta)</string>
```
Comment (moving-bits) [inline main/src/main/res/values/strings.xml:3016]: This triggers a question (not directly related to this PR): Shall we remove "beta" status from UnifiedMap in current nightly version?

Comment (eddiemuc) [reply main/src/main/res/values/strings.xml]: I have no opinion about that. I changed the PR such that this question can be answered separately (and, if need be, changed in a separate PR)

**Comment by ztNFny** [comment]
How about showing WIG only if UnifiedMap is enabled? Like to change the feature switch to return the map type

**Comment by eddiemuc** [comment]
> How about showing WIG only if UnifiedMap is enabled? Like to change the feature switch to return the map type

WIG and unified map are unrelated in all but the display of mapping zones. It is perfectly possible tomolay a wherigo w/o map usage. This I don't want to connect the two in this way.

On the contrary, maybe we get a few more users to switch to unified map with current approach

**Comment by eddiemuc** [comment]
BTW there's no feature switch any more for wherigo. It is enabled for all users.

**Pull request merged**